Server/server/models/votes.js

```diff
@@ -0,0 +1,33 @@
+
+module.exports = (sequelize, DataTypes) => {
```

Comment:
So I wouldn't want to see any `module.exports` try using `exports defaults` instead.

Reply:
Done this, it only exists in the sequelizerc file now, that one was not changed because it was auto- generated, I am not sure if it will still work if it is changed